cli/src/cmd_net.rs

```diff
@@ -0,0 +1,1485 @@
+                        oxide::types::IpNet::V4(a) => a.to_string(),
+                        oxide::types::IpNet::V6(a) => a.to_string(),
+                    };
+
+                    let alb = addr_lot_blocks
+                        .iter()
+                        .find(|x| x.1.id == a.address_lot_block_id)
+                        .unwrap();
+
+                    writeln!(&mut tw, "{}\t{}", addr, *alb.0.name)?;
+                }
+                tw.flush()?;
+                println!();
+
+                writeln!(
+                    &mut tw,
+                    "{}\t{}\t{}\t{}\t{}\t{}\t{}\t{}\t{}\t{}\t{}\t{}\t{}\t{}\t{}\t{}\t{}",
+                    "BGP Peer".dimmed(),
+                    "Config".dimmed(),
+                    "Export".dimmed(),
+                    "Import".dimmed(),
+                    "Communities".dimmed(),
+                    "Connect Retry".dimmed(),
+                    "Delay Open".dimmed(),
+                    "Enforce First AS".dimmed(),
+                    "Hold Time".dimmed(),
+                    "Idle Hold Time".dimmed(),
+                    "Keepalive".dimmed(),
+                    "Local Pref".dimmed(),
+                    "Md5 Auth".dimmed(),
+                    "Min TTL".dimmed(),
+                    "MED".dimmed(),
+                    "Remote ASN".dimmed(),
+                    "VLAN".dimmed(),
+                )?;
+                for p in &config.bgp_peers {
+                    writeln!(
+                        &mut tw,
+                        "{}\t{}\t{:?}\t{:?}\t{:?}\t{}\t{}\t{}\t{}\t{}\t{}\t{:?}\t{:?}\t{:?}\t{:?}\t{:?}\t{:?}",
+                        p.addr,
+                        match &p.bgp_config {
+                            NameOrId::Id(id) =>  bgp_configs[id].to_string(),
+                            NameOrId::Name(name) => name.to_string(),
+                        },
+                        p.allowed_export,
+                        p.allowed_import,
+                        p.communities,
+                        p.connect_retry,
+                        p.delay_open,
+                        p.enforce_first_as,
+                        p.hold_time,
+                        p.idle_hold_time,
+                        p.keepalive,
+                        p.local_pref,
+                        p.md5_auth_key,
+                        p.min_ttl,
+                        p.multi_exit_discriminator,
+                        p.remote_asn,
+                        p.vlan_id,
+                    )?;
+                }
+                tw.flush()?;
+                println!();
+
+                // Uncomment to see full payload
+                //println!("");
+                //println!("{:#?}", config);
+                //println!("");
+            }
+        }
+
+        Ok(())
+    }
+}
+
+/// Get the status of BGP.
+#[derive(Parser, Debug, Clone)]
+#[command(verbatim_doc_comment)]
+#[command(name = "net bgp status")]
+pub struct CmdBgpStatus {}
```

Comment:
The only difference is the output is formatted a bit nicer. Seems like this would fit in better as a flag to `system networking bgp status`? Something like `--output table`? This is likely true of several commands in this PR.

Reply:
```
ry@masaka:~$ oxide net bgp status
switch0
=======
Peer Address  Local ASN  Remote ASN  Session State  State Duration
169.254.30.1  65547      64502       Established    1h 5m 33s 942ms
169.254.10.1  65547      64500       Established    1h 5m 38s 910ms

switch1
=======
Peer Address  Local ASN  Remote ASN  Session State  State Duration
169.254.20.1  65547      64500       Established    1h 5m 38s 855ms
169.254.40.1  65547      64502       Established    1h 5m 33s 888ms
```
```
ry@masaka:~$ oxide system networking bgp status
[
  {
    "addr": "169.254.30.1",
    "local_asn": 65547,
    "remote_asn": 64502,
    "state": "established",
    "state_duration_millis": 3943140,
    "switch": "switch0"
  },
  {
    "addr": "169.254.10.1",
    "local_asn": 65547,
    "remote_asn": 64500,
    "state": "established",
    "state_duration_millis": 3948107,
    "switch": "switch0"
  },
  {
    "addr": "169.254.20.1",
    "local_asn": 65547,
    "remote_asn": 64500,
    "state": "established",
    "state_duration_millis": 3948072,
    "switch": "switch1"
  },
  {
    "addr": "169.254.40.1",
    "local_asn": 65547,
    "remote_asn": 64502,
    "state": "established",
    "state_duration_millis": 3943104,
    "switch": "switch1"
  }
]
```

Reply:
For the moment, opting to have `show` commands that display information in a more human-oriented way than their `status` and `view` counterparts.